Makefile

```diff
@@ -27,15 +27,18 @@ rounduptest:
 flake8:
 	flake8 scripts/
 
+python_unittest:
+	python -m unittest discover -v -s scripts/lib/libvirt/
+
 # for travis-CI:
 install: debianinstall genericinstall
 
 debianinstall:
 	sudo apt-get update -qq
-	sudo apt-get -y install libxml-libxml-perl libjson-xs-perl
+	sudo apt-get -y install libxml-libxml-perl libjson-xs-perl python-libvirt
 
 suseinstall:
-	sudo zypper install perl-JSON-XS perl-libxml-perl
+	sudo zypper install perl-JSON-XS perl-libxml-perl libvirt-python
 
 genericinstall:
 	sudo pip install bashate flake8
```

Comment:
Actually I find this ugly.  There are installations via zypper and others via pip as root.  I would never install anything with pip as root in my system. I would suggest two options here:

* Because python-lake8 is in Factory, so I would submit python-bashate from Cloud:OpenStack:Master for Factory too.
* and/or use virtualenv to install all the python dependencies that are not in any repository.